src/components/Pill/Pill.test.tsx

```diff
@@ -19,14 +19,30 @@ test('it renders a Pill with a label', async () => {
   expect(root).toBeInTheDocument();
 });
 
-test('it renders a Pill with the provided "color"', async () => {
```

Comment:
I'm curious - did you used to be able to define whatever color you wanted and now we're providing a limited set of available colors by concern? I'm new to Chroma - want to make sure I'm on the same page for when I start working in here.

Reply:
`default` and `primary` were the only color options prior to this. If a custom color is needed, the only way to do that is through a custom class.